src/lines.ts

```diff
@@ -215,7 +216,10 @@ export class Lines extends Base<ILinesSettings> {
       for (let yOffset = -weight; yOffset < weight; yOffset += 0.5) {
         for (let xOffset = -weight; xOffset < weight; xOffset += 0.5) {
           // -- set base matrix to translate canvas pixel coordinates -> webgl coordinates
-          mapMatrix.translateMatrix(-offset.x + (xOffset / scale), -offset.y + (yOffset / scale));
+          mapMatrix
+            .setSize(canvas.width, canvas.height)
```

Comment:
I don't believe we need to re-set the size, as it wouldn't have changed between when it is set above and here.

Reply:
The same for scale, it wouldn't have changed as well.

Reply:
No, the size and scale has not changed. But if you do not reset them, then each translateMatrix() in the loop is appended to the previous matrix. These webgl matrices are not my expertice, but have a look at item 12-15 in this output of mapMatrix.

Before:
![glify_mapmatrix_before](https://user-images.githubusercontent.com/35254400/102637582-7de3b500-4156-11eb-8fe5-6e665cdd270c.png)

After my change:
![glify_mapmatrix_after](https://user-images.githubusercontent.com/35254400/102637610-8a680d80-4156-11eb-9a2d-2eb5a63014c6.png)

Running the example I can see that this fixes the problem for me, but there may be better ways of fixing it.